test.js

```diff
@@ -1,5 +1,9 @@
 var t = "hello";
 
-if (t === "hello') {
+if (t === "hello") {
     console.log('ok');
 }
+
+var donothing = false;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["donothing" is defined but never used](https://www.codacy.com/app/rubstar/rvs-pr-testing/file/2496536173/issues/source?bid=3260975&fileBranchId=3269370#l7)